src/Versity.Users/Tests/Users.Tests/Application/Behaviors/LoggingPipelineBehaviorTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Application.Behaviors;
+using Application.Exceptions;
+using Application.RequestHandlers.Auth.Commands.ConfirmEmail;
+using FluentAssertions;
+using FluentValidation;
+using MediatR;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Logging;
+using Moq;
+
+namespace Users.Tests.Application.Behaviors;
+
+public class LoggingPipelineBehaviorTests
+{
+    private readonly Mock<ILogger<LoggingPipelineBehavior<ConfirmEmailCommand, IdentityResult>>> _loggerMock;
+
+    public LoggingPipelineBehaviorTests()
+    {
+        _loggerMock = new Mock<ILogger<LoggingPipelineBehavior<ConfirmEmailCommand, IdentityResult>>>();
+    }
+    
+    [Fact]
+    public async Task ValidationPipelineBehavior_ShouldInvokeNextPipeline_WhenDelegateIsValid()
```

Comment:
i guess it should be LoggingPipelineBehavior_ShouldInvokeNextPipeline_WhenDelegateIsValid